src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import duke.exceptions.EmptyCommandException;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void response(String line, TaskManager action) {
+        String[] words = line.split(" ");
+        String commandWord = words[0];
```

Comment:
Avoid magic numbers. Perhaps you can consider putting the numbers into a descriptive variable (e.g. `0` can be put into `INDEX_OF_COMMAND_WORD`)